boards/arm64/qemu_cortex_a55/doc/index.rst

```diff
@@ -0,0 +1,100 @@
+.. _qemu_cortex_a55:
+
+ARM Cortex-A55 Emulation (QEMU)
+###############################
+
+Overview
+********
+
+This board configuration will use QEMU to emulate a generic Cortex-A55 hardware
+platform.
+
+This configuration provides support for an ARM Cortex-A55 CPU and these
+devices:
+
+* GIC-400 interrupt controller
+* ARM architected timer
+* PL011 UART controller
+
+Hardware
+********
+Supported Features
+==================
+
+The following hardware features are supported:
+
++--------------+------------+----------------------+
+| Interface    | Controller | Driver/Component     |
++==============+============+======================+
+| GIC          | on-chip    | interrupt controller |
++--------------+------------+----------------------+
+| PL011 UART   | on-chip    | serial port          |
++--------------+------------+----------------------+
+| ARM TIMER    | on-chip    | system clock         |
++--------------+------------+----------------------+
+
+The kernel currently does not support other hardware features on this platform.
+
+Devices
+========
+System Clock
+------------
+
+This board configuration uses a system clock frequency of 62.5 MHz.
+
+Serial Port
+-----------
+
+This board configuration uses a single serial communication channel with the
+CPU's UART0.
+
+Known Problems or Limitations
+==============================
+
+The following platform features are unsupported:
+
+* Writing to the hardware's flash memory
+
+
+Programming and Debugging
+*************************
+
+Use this configuration to run basic Zephyr applications and kernel tests in the QEMU
+emulated environment, for example, with the :ref:`synchronization_sample`:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/synchronization
+   :host-os: unix
+   :board: qemu_cortex_a55
+   :goals: run
+
+This will build an image with the synchronization sample app, boot it using
+QEMU, and display the following console output:
+
+.. code-block:: console
+
+	[0/1] To exit from QEMU enter: 'CTRL+a, x'[QEMU] CPU: cortex-a55
+	*** Booting Zephyr OS build zephyr-v3.2.0-2393-g0aef1450f031 ***
+	thread_a: Hello World from cpu 0 on qemu_cortex_a55!
```

Comment:
Just for me to understand - why is printing the same thing 3 times?

Reply:
The printing goes on forever, it's not just the lines included in the documentation. This is just a sample (more exactly: samples/synchronization) meant to do a basic sanity check (or so I believe). Also, the output is taken from the non-SMP dts.